x-pack/plugins/apm/server/routes/alerts/rule_types/anomaly/register_anomaly_rule_type.test.ts

```diff
@@ -151,6 +167,7 @@ describe('Transaction duration anomaly alert', () => {
                             job_id: '1',
                             partition_field_value: 'foo',
                             by_field_value: 'type-foo',
+                            detector_index: 0, // latency index
```

Comment:
Perhaps more readable. This way you won't need the comment, and we exercise the helper method
```suggestion
                            detector_index: getApmMlDetectorIndex(ApmMlDetectorType.txLatency)
```